dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -147,8 +147,19 @@ func buildModules(srcPath, name, ufsType, moduleFlag, version string, modules ma
 		for _, arg := range strings.Split(moduleEntry.mavenArgs, " ") {
 			moduleMvnArgs = append(moduleMvnArgs, arg)
 		}
+		var versionMvnArg string
+		for _, arg := range strings.Split(moduleEntry.mavenArgs, " ") {
```

Comment:
should this loop over `moduleMvnArgs` instead, in case `ufs.hadoop.version` is set in `mvnArgs`?

Reply:
fixed